### PR TITLE
Link on promotions list

### PR DESCRIPTION
**Pull request by vlaux**
#### What is the purpose of this pull request?
[CH31497](https://app.clubhouse.io/vtex/story/31497/enable-open-in-new-tab-link-on-promotions-list)

#### What problem is this solving?
A user should be able to right click a promotion in the list and use the "Open in new tab" option, as well as ctrl (cmd) + click a row to open in new tab (basically it should really work as a link).
Since we plan to migrate to Table v2, we agreed that it is enough to have this only on the name column.

#### How should this be manually tested?
[Running workspace](https://vlauxbeta--pricingqa.myvtex.com/admin/promotions)

#### Screenshots or example usage
image

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/__mocks__/vtex.render-runtime.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export const Link = ({ children }) => <a href="dummy">{children}</a>
```
Comment (ericreis) [inline react/__mocks__/vtex.render-runtime.js]: Why do you need this?

File: react/styles.global.css [hunk 2]
```diff
@@ -6,3 +6,6 @@ body,
 .render-provider {
   height: 100%;
 }
+.c-inherit {
```
Comment (ericreis) [inline react/styles.global.css]: Why do you need this?

File: react/__mocks__/vtex.render-runtime.js (hunk 1 above)
Comment (vlaux) [reply react/__mocks__/vtex.render-runtime.js]: For the tests. They will break otherwise.

File: react/styles.global.css (hunk 2 above)
Comment (vlaux) [reply react/styles.global.css]: To fix the link's color

**Pull request merged**